omniduct/databases/hiveserver2.py

```diff
@@ -277,7 +277,6 @@ def _dataframe_to_table(
         partition = partition or {}
         table_props = table_props or {}
         dtype_overrides = dtype_overrides or {}
-
```

Comment:
This whitespace should probably be left in, helping the eye to find blocks of related code.